how-to/customize-workspace/client/src/auth-shapes.ts

```diff
@@ -0,0 +1,27 @@
+export interface AuthProvider {
+	init: <T>(options: T) => Promise<void>;
```

Comment:
All these method definitions look a bit odd, they should have the `: ` in them and the `=>` should be `: `